projects/eslint-config/plugins/portal/lib/rules/no-metal-plugins.js

```diff
@@ -44,22 +54,42 @@ module.exports = {
 		},
 		fixable: null,
 		messages: {
+			'no-metal-affix':
+				'metal-affix is deprecated; use native offset instead',
 			'no-metal-ajax':
 				"metal-ajax is deprecated; use `import {fetch} from 'frontend-js-web';` instead",
+			'no-metal-anim':
+				'metal-anim is deprecated; use native `window.requestAnimationFrame()` instead',
 			'no-metal-aop':
 				"metal-aop is deprecated; use `import {AOP} from 'frontend-js-web';` instead",
+			'no-metal-assertions':
+				'metal-assertions is deprecated; use native `typeof` instead',
 			'no-metal-clipboard':
 				'metal-clipboard is deprecated; use ClipboardJS instead',
 			'no-metal-debounce':
 				"metal-debounce is deprecated; use `import {debounce} from 'frontend-js-web';` instead",
+			'no-metal-dom':
+				'metal-dom is deprecated; see https://issues.liferay.com/browse/LPS-122383 to learn what to use instead',
+			'no-metal-key':
+				'metal-key is deprecated; use native key event handling instead',
 			'no-metal-keyboard-focus':
-				"metal-keyboard-focus is deprecated; use `import {debounce} from 'frontend-js-web';` instead",
+				'metal-keyboard-focus is deprecated; use native focus behaviour instead',
+			'no-metal-multimap':
+				'metal-multimap is deprecated; use native Set instead',
```

Comment:
This seems wrong. A `Set` isn't an appropriate replacement (it only holds values); I think you mean `Map` (which holds key/value pairs).